types/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export function HomebaseProvider(props?: object): object;
+export function useTransact(): Array<Array<object>>;
```

Comment:
Can we type the transact function as tacking a transaction?
Then useTransact can return a Tuple with the first element being a transact function

Reply:
> Can we type the transact function as tacking a transaction?
Do you mean s/tacking/taking?

Reply:
yes, taking